### ISSUE TITLE
Failed sending transaction using old version of GUI

### ISSUE THREAD

**Issue by abundantone** [monero-project/monero#5354]
Hi, I tried to send 345 XMR from the GUI to an exchange about 8 hours ago.  Only later did I realize that the GUI was quite outdated (v0.11.1.0).  The balance in the GUI now reads zero but the transaction status shows as 'PENDING' and the exchange shows no deposit.   I also failed to take into consideration the recent hard-fork.  Any suggestions on how I can correct this?  Much appreciated! 

<img width="958" alt="Screen Shot 2019-03-26 at 8 48 39 AM" src="https://user-images.githubusercontent.com/30573590/54998346-4e654000-4fa4-11e9-94df-2d242639bd64.png">

**Comment by moneromooo-monero** [comment]
Quit the wallet, flush_txpool in monerod, run the wallet again, wait for it to refresh a couple times and realize the tx is gone and not mined.

**Comment by abundantone** [comment]
> Quit the wallet, flush_txpool in monerod, run the wallet again, wait for it to refresh a couple times and realize the tx is gone and not mined.

Thanks for the super quick reply.

Once I do that, I assume that I should update the GUI prior to attempting to send again?

Is there anything that I need to do regarding the recent hard fork?

**Comment by moneromooo-monero** [comment]
Yes, good point, run the latest release when starting again :)
That's the only thing you should need to do.

**Comment by dEBRUYNE-1** [comment]
You can resolve your issue with these guides:

https://monero.stackexchange.com/questions/7989/i-forgot-to-upgrade-from-cli-or-gui-v0-13-to-cli-or-gui-v0-14-and-as-a-result

https://monero.stackexchange.com/questions/7993/i-forgot-to-upgrade-from-cli-or-gui-v0-13-to-cli-or-gui-v0-14-and-created-pe

**Comment by abundantone** [comment]
Hi,

Thank you so much for the guides.

I just have one question:

Do I need to upgrade to GUI v0.14.0 *first* and then use the following
guide to correct the blockchain?
https://monero.stackexchange.com/questions/7993/i-forgot-to-upgrade-from-cli-or-gui-v0-13-to-cli-or-gui-v0-14-and-created-pe

Or, do I follow that guide and correct the old GUI v0.11.1.0 *first *and
then upgrade to the new GUI v0.14.0?

Thanks so much in advance!

Jeffrey

**Comment by abundantone** [comment]
> Quit the wallet, flush_txpool in monerod, run the wallet again, wait for it to refresh a couple times and realize the tx is gone and not mined.

Hi,

Thank you so much for the guides.  

I just have one question:

Do I need to upgrade to GUI v0.14.0 first and then use the following guide to correct the blockchain?  
https://monero.stackexchange.com/questions/7993/i-forgot-to-upgrade-from-cli-or-gui-v0-13-to-cli-or-gui-v0-14-and-created-pe

Or, do I follow that guide to correct the blockchain using my old GUI v0.11.1.0 first and then upgrade to the new GUI v0.14.0?

Thanks so much in advance!

**Comment by dEBRUYNE-1** [comment]
@abundantone - You first need to follow this guide:

https://monero.stackexchange.com/questions/7989/i-forgot-to-upgrade-from-cli-or-gui-v0-13-to-cli-or-gui-v0-14-and-as-a-result

Which means you have to upgrade to GUI v0.14.0.0. Please read aforementioned guide diligently, as there are some remarks on, for instance, the database conversion. Once GUI v0.14.0.0 is fully synced, you can apply this guide to recover your 'lost' transaction:

https://monero.stackexchange.com/questions/7993/i-forgot-to-upgrade-from-cli-or-gui-v0-13-to-cli-or-gui-v0-14-and-created-pe

**Comment by abundantone** [comment]
Hi,

Sorry for so many questions :-/

I followed the first guide and downloaded GUI v0.14.0.0 and have attempted
to start the daemon several times but I keep getting the same error message:

[image: Screen Shot 2019-03-26 at 5.11.45 PM.png]

So, I click OK and I click on the 'Start daemon' button from the following
screen.

[image: Screen Shot 2019-03-26 at 5.08.59 PM.png]

But after a few minutes, this cycle repeats.

Is this normal?

With great appreciation.

J

**Comment by abundantone** [comment]
[image: Screen Shot 2019-03-26 at 6.27.41 PM.png]

Jeffrey Lee

**Comment by dEBRUYNE-1** [comment]
The release thread of the GUI states:

>This version, if using your own (local) node, requires a database conversion, which may take 5-10 minutes and the GUI will show that it's Disconnected (or unable to connect) from the daemon (monerod). I'd advise to simply let it run and after the database conversion has completed the GUI will connect back to the daemon (monerod).

Given that you have to perform two database conversions, it will take significantly longer. Hence my previous comment of "Please read aforementioned guide diligently, as there are some remarks on, for instance, the database conversion.".

**Comment by abundantone** [comment]
The updated GUI is now in the process of synchronizing.   The confusing
part is that I kept getting the error message of the the daemon failing to
start and requesting my password again and again.   But it finally seems to
be progressing.

Thanks so much for your continuing support and patience.

Jeffrey Lee

**Comment by abundantone** [comment]
Hi,

The GUI has synchronized however it is stuck ->  the 'Wallet Blocks
Remaining' metric is stuck at 330170.

Do you have any suggestions?

Thanks!

Jeffrey
[image: Screen Shot 2019-03-27 at 6.40.45 PM.png]

**Comment by abundantone** [comment]
I just downloaded Daedalus v0.13.1#3.0.1.5130 and I have the same problem.

Jeffrey

**Comment by abundantone** [comment]
Oops - sorry, please disregard the last email re Daedalus GUI.

The syncing of the Monero GUI seems to be making progress, albeit very
slowly.

Thanks for all your help.
[image: Screen Shot 2019-03-28 at 2.56.28 AM.png]

.Jeffrey

**Comment by dEBRUYNE-1** [comment]
If the wallet refresh (i.e. "Wallet blocks remaining") is not progressing anymore, try restarting the GUI.

**Comment by dEBRUYNE-1** [comment]
Good to hear! Going to mark this as resolved then.

**Comment by dEBRUYNE-1** [comment]
+resolved

**Comment by abundantone** [comment]
I think I need more help.

I started to follow guide:
https://monero.stackexchange.com/questions/7993/i-forgot-to-upgrade-from-cli-or-gui-v0-13-to-cli-or-gui-v0-14-and-created-pe

I was able to flush the tx pool and the balance restored in the GUI.

I exited the GUI (leaving the Daemon running) and renamed the wallet in the
directory.

When I re-started the GUI, the GUI did not recognize my computer:

[image: Screen Shot 2019-03-28 at 9.15.08 AM.png]
[image: Screen Shot 2019-03-28 at 9.24.45 AM.png]
[image: Screen Shot 2019-03-28 at 9.44.53 AM.png]
[image: Screen Shot 2019-03-28 at 9.30.42 AM.png]


Is this what should have happened?

What mode should I select?  Simple mode or advanced mode?

One other question concerns the location of the old and new versions of the
GUI:

[image: Screen Shot 2019-03-28 at 9.47.51 AM.png]

The new version of the GUI v0.14.0.0 got stored in a folder on my desktop
instead of replacing the old version of the GUI which is stored in
'Applications' folder.  Can I / Should I delete the old version(s) of the
GUI in the 'Applications' folder and move the new GUI into the
'Applications' folder?

Thanks once again!

Jeffrey
